src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/DataFrameTests.cs

```diff
@@ -404,6 +404,22 @@ public void TestSignaturesV2_3_X()
             Assert.IsType<RelationalGroupedDataset>(_df.GroupBy(_df["age"]));
             Assert.IsType<RelationalGroupedDataset>(_df.GroupBy(_df["age"], _df["name"]));
 
+            Assert.IsType<DataFrame>(_df.GroupBy("name").Mean("age"));
+            Assert.IsType<DataFrame>(
+                _df.WithColumn("tempAge", _df["age"]).GroupBy("name").Mean("age", "tempAge"));

```

Comment:
for these GroupBy.fn tests, let's pull out _df.WithColumn("tempAge", _df["age"]).GroupBy("name") into a var within it's own scope, { ... } and reuse.

Reply:
Same with _df.GroupBy("name") as well.

Reply:
I have made the changes. Please let me know if the var name makes sense. Thanks!